### PR TITLE
created identify button for waysofbeingindex

### PR DESCRIPTION
**Pull request by Zmuse001**
### Changes
created identify button for waysofbeingindex and linked it to identification form

### Link to Issue 
https://app.asana.com/0/1206510424537042/1207090158804667/f

### Comments, Notes, or  Questions
This section is if you want to share something related to the PR with the reviewer. 

### Related Screenshots 
Screenshot 2024-05-24 at 4 34 28 PM

### Before you submit for review, Did you...

- [ ] Added relevant tests (if applicable)
- [ ] Linked Asana ticket to PR
- [ ] Added comments, notes, or questions for reviewwer
- [ ] Added relevant screenshots
- [ ] Ensured ticket title is clear and concise(i.e. 'Donation Form Feature' instead of 'sfg/chels/donation-form'

### CODE REVIEW

**Comment by crestu** [comment]
Screenshot 2024-05-28 at 12 19 45 PM
im seeing this error when i click the identify button

**Comment by ridwvn22** [comment]
> Screenshot 2024-05-28 at 12 19 45 PM im seeing this error when i click the identify button

I am also seeing this error.

File: app/views/ways_of_being/index.html.erb [hunk 1]
```diff
@@ -3,8 +3,12 @@
         <h1 class="text-center font-bold lg:text-2xl py-4 border-b border-gray-600">Behaviors</h1>
         <ul class="px-4">
             <% @ways_of_being.each do |way_of_being| %>
-            <li class="bg-white py-2 border-b border-gray-400 text-gray-400">
-                <%= way_of_being.name %>
+            <li class="flex w-full justify-between bg-white py-2 border-b border-gray-400 text-gray-400">
+                <div><%= way_of_being.name %></div>
+                <%= link_to new_identification_path(student_way_of_being_id: way_of_being.id), class: "strategy-btn border-2 border-gray-600 hover:bg-green-600 text-black px-2 rounded flex items-center space-x-1 text-lg" do %>
```
Comment (edithemmings) [inline app/views/ways_of_being/index.html.erb:8]: Needs to be the student way of being id -- not the way of being id

Comment (Zmuse001) [reply app/views/ways_of_being/index.html.erb:8]: it doesn't work if i do that. its a huge error

**Comment by crestu** [comment]
Screenshot 2024-05-29 at 11 43 11 AM
im getting this error after clicking submit on the way of being page.

**Comment by edithemmings** [comment]
Waiting to review until that bug with student way of being id vs way of being id is resolved

**Comment by crestu** [comment]
Screenshot 2024-06-04 at 3 23 25 PM

im seeing duplicates of each way of being

File: app/views/ways_of_being/index.html.erb [hunk 2]
```diff
@@ -2,7 +2,12 @@
     <div class="container bg-white h-screen w-64 rounded-lg">
         <h1 class="text-center font-bold lg:text-2xl py-4 border-b border-gray-600">Behaviors</h1>
         <ul class="px-4">
-            <% @ways_of_being.each do |way_of_being| %>
```
Comment (edithemmings) [inline app/views/ways_of_being/index.html.erb:5]: The loop got lost

File: app/views/ways_of_being/index.html.erb [hunk 3]
```diff
@@ -2,7 +2,12 @@
...
+            <li class="flex w-full justify-between bg-white py-2 border-b border-gray-400 text-gray-400">
+                <div><%= way_of_being.name %></div>
+                <%= link_to new_identification_path(student_way_of_being_id: way_of_being.id), class: "strategy-btn border-2 border-gray-600 hover:bg-green-600 text-black px-2 rounded flex items-center space-x-1 text-lg" do %>
```
Comment (edithemmings) [inline app/views/ways_of_being/index.html.erb:7]: Still passing a way of being id into a student way of being id

**Pull request closed without merging**